### DISCUSSION TITLE
REMOVE UBRS/LBRS TUNING + Thoughts on MC

### DISCUSSION THREAD

**Discussion by Spook9** [chromiecraft/chromiecraft#3411] [Suggestions]
Havent we had them tuned long enough? 
UBRS is already doable with just 3 ppl and some patience. 

The tuning at this point just isnt serving any purpose imo. If ppl will end up soloing it will still require skill. On the contrary it has some side effects like the fact that LBRS is unsoloable for when entering you are still slapped by the tuning.

Please also start considering the same for MC moving forward. If not a flat out removal of the tuning something similar where it just wont take as long as a time to complete for its already hitting the nerves of some ppl and very brutal for most pugs with a lack of healers or well geared/experienced tanks. 
Maybe even consider unlocking it for wbuffs so people who want can have some fun pushing meters or just blasting in MC. Just a thought

Hello to Temper :3
and thank you for reading
-Spook

**Comment by heyitsbench** [comment]
I am of the opinion that once the dark portal is open, then the tuning should be removed on the vanilla instances.

**Comment by Neversong97** [comment]
remove tuning from older raids and make actual raid "tuned" etc
